includes/Core/User_Input/User_Input.php

```diff
@@ -92,7 +93,10 @@ public function __construct(
 		$this->site_specific_answers = new Site_Specific_Answers( $options ?: new Options( $context ) );
 		$this->user_options          = $user_options ?: new User_Options( $context );
 		$this->user_specific_answers = new User_Specific_Answers( $this->user_options );
-		$this->rest_controller       = new REST_User_Input_Controller( $this );
+		$this->rest_controller       = new REST_User_Input_Controller(
+			$this,
+			new Survey_Queue( $this->user_options )
```

Comment:
We have the `Survey_Queue` instance already instantiated in the `User_Surveys` class. Instead of creating a new instance here, we should get the survey queue instance from that class and pass it into this controller.

Reply:
Updated, thank you for the directions!